process_scenario_weo_2024.R

```diff
@@ -87,11 +76,21 @@ weo_2024_fig_chptr_3_raw <-
   )
 
 iea_global_ev_2024_raw <-
-  readr::read_csv(
-    file = iea_global_ev_2024_raw_full_filepath,
-    show_col_types = FALSE
+  read_xlsx(
+    path = iea_global_ev_2024_raw_full_filepath,
+    sheet = "electric-vehicle-sales-by-regio"
```

Comment:
is it intentional that the sheet name ends with "regio" not "region"?

Reply:
sheet name is truncated when uploading data from IEA website, so sheet name title will be electric-vehicle-sales-by-regio